addons/t9n/models/language.py

```diff
@@ -4,3 +4,5 @@
 class Language(models.Model):
     _name = "t9n.language"
     _description = "Language"
+
+    name = fields.Char()
```

Comment:
A few things:
- We probably also want the language name to be required (otherwise we can have "Unnamed" languages).
- It may be clearer if we give it a string so it looks better in the views
- The `, translate=True` isn't necessary for now, but at some point we will probably pre-populate some languages for the db (i.e. `data` file records so users don't have to manually add standard languages). The `translate=True` makes it so these records will be included in the terms to be translated (i.e. so we can make sure that the language shows up in the current language, although we don't currently do that so maybe we won't do that?)

```suggestion
    name = fields.Char('Language', required=True, translate=True)
```